src/jit/optcse.cpp

```diff
@@ -882,37 +970,68 @@ class CSE_DataFlow
 {
     BitVecTraits* m_pBitVecTraits;
     EXPSET_TP     m_preMergeOut;
+#ifdef DEBUG
+    Compiler* m_pCompiler;
+#endif // DEBUG
 
 public:
     CSE_DataFlow(Compiler* pCompiler) : m_pBitVecTraits(pCompiler->cseTraits), m_preMergeOut(BitVecOps::UninitVal())
     {
+#ifdef DEBUG
+        m_pCompiler = pCompiler;
```

Comment:
Constructors should initialize data members in the initialization list.

Reply:
OK